nullaway/src/test/java/com/uber/nullaway/VarargsTests.java

```diff
@@ -45,14 +46,41 @@ public void testNullableVarargs() {
             "import javax.annotation.Nullable;",
             "public class Utilities {",
             " public static String takesNullableVarargs(Object o, @Nullable Object... others) {",
-            "  String s = o.toString() + \" \";",
-            "  // BUG: Diagnostic contains: enhanced-for expression others is @Nullable", // Shouldn't be an error!
-            "  for (Object other : others) {",
-            "    s += (other == null) ? \"(null) \" : other.toString() + \" \";",
+            "  String s = o.toString() + \" \" + others.toString();",
+            "  return s;",
+            " }",
+            "}")
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "public class Test {",
+            "  public void testNonNullVarargs(Object o1, Object o2, Object o3, @Nullable Object o4) {",
+            "    Utilities.takesNullableVarargs(o1, o2, o3, o4);",
+            "    Utilities.takesNullableVarargs(o1);", // Empty var args passed
+            "    Utilities.takesNullableVarargs(o1, o4);",
+            "    Utilities.takesNullableVarargs(o1, (java.lang.Object) null);",
+            "    Object[] x = null;",
+            "    // BUG: Diagnostic contains: passing @Nullable parameter 'x'",
+            "    Utilities.takesNullableVarargs(o1, x);",
```

Comment:
Is this `null` array case missing from `testNonNullVarargs` above? (And in general for legacy and others too?)

Reply:
Yup, added in 71301a386e6285716f9a8d07c62fe62ce8a476ab